language/examples/bqml/bqml_llm_examples.ipynb

```diff
@@ -0,0 +1,1114 @@
+{
```

Comment:
<p>Could the title be something more like:</p><p>Intro to using BigQuery with Vertex AI LLMs</p><p> </p><p>to be a bit more descriptive without over-using acronyms?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/127/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>